dev/sbt-checkstyle

```diff
@@ -26,7 +26,8 @@ ERRORS=$(echo -e "q\n" \
         -Pkubernetes \
         -Pyarn \
         -Phive \
-        -Phive-thriftserver \
+        -Phadoop-3.1 \
```

Comment:
@wangyum, please go ahead for a separate PR. I will review.

Reply:
OK.